src/main/kotlin/io/curity/identityserver/plugin/frejaeid/authentication/RequestLogicHelper.kt

```diff
@@ -223,6 +223,18 @@ class RequestLogicHelper(private val config: FrejaEidAuthenticatorPluginConfig)
         return "frejaeid://bindUserToTransaction?transactionReference=$authRef"
     }
 
+    private fun buildJsonAuthRequest(postData: Map<String, Any>): String
```

Comment:
I think we should fix this in the impl of the JSON service by adding an overload that allows this client (and others) to indicate that they don't want HTML escaping.